drivers/spi/spi_ll_stm32.c

```diff
@@ -938,6 +938,15 @@ static void spi_stm32_irq_config_func_##id(const struct device *dev)		\
 #define STM32_SPI_USE_SUBGHZSPI_NSS_CONFIG(id)
 #endif
 
+#if !defined(CONFIG_SOC_SERIES_STM32F1X) \
+	&& (!defined(CONFIG_SOC_SERIES_STM32L1X) || defined(SPI_CR2_FRF))
+#define STM32_SPI_USE_FRAME_FORMAT(id)				\
+	DT_INST_PROP_OR(id, frame_format, SPI_FRAME_FORMAT_TI)	\
```

Comment:
You should add explicit comparison with 
`DT_INST_PROP_OR(...) == SPI_FRAME_FORMAT_TI ? ...`

Reply:
changed to have SPI_FRAME_FORMAT_MOTOROLA as default